src/main/java/org/zalando/problem/spring/web/advice/validation/ConstraintViolationProblem.java

```diff
@@ -38,16 +39,16 @@
     public static final URI CONSTRAINT_VIOLATION = URI.create(CONSTRAINT_VIOLATION_VALUE);
 
     private final Optional<String> detail;
-    private final ImmutableList<Violation> violations;
+    private final List<Violation> violations;
 
-    public ConstraintViolationProblem(final ImmutableList<Violation> violations) {
+    public ConstraintViolationProblem(final List<Violation> violations) {
```

Comment:
what do you think about such implementation of copying https://www.jayway.com/2013/11/12/immutable-list-collector-in-java-8/